src/components/GeneralModal/GeneralModal.jsx

```diff
@@ -23,14 +31,24 @@ const GeneralModal = ({ openModal, setOpenModal, title, message }) => {
           <div className={classes.paper}>
             <h2>{title}</h2>
             <p>{message}</p>
-            <Button
-              className={classes.buttonGrad}
-              classvariant="contained"
-              type="button"
-              onClick={() => setOpenModal(false)}
-            >
-              cerrar
-            </Button>
+            <div className={classes.buttonsModal}>
+              <Button
```

Comment:
This button must be a conditional render, because this component is use in differentes parts of the aplication

Reply:
The buttons are now rendered as a children, doesn't have to be conditioned